src/api/mod.rs

```diff
@@ -223,6 +223,7 @@ impl Api {
             .add_root_certificate(cert_admin_api)
             .add_root_certificate(cert_peridio)
             .add_root_certificate(cert_nerveshub)
+            .danger_accept_invalid_certs(true)
```

Comment:
I imagine this was used for testing but lets ensure its removed prior to merging.